cabot/cabot_config.py

```diff
@@ -5,6 +5,8 @@
 GRAPHITE_PASS = os.environ.get('GRAPHITE_PASS')
 GRAPHITE_FROM = os.getenv('GRAPHITE_FROM', '-10minute')
 JENKINS_API = os.environ.get('JENKINS_API')
+if not JENKINS_API.endswith('/'):
```

Comment:
Should we do it inside the code? In the Jenkins check ... (I am not a fan of placing code in config files - even though I am guilty of such crimes)